person.rb

```diff
@@ -1,9 +1,9 @@
-require_relative 'nameable'
-require_relative 'capitalize_decorator'
-require_relative 'trimmer_decorator'
+require_relative './nameable'
+require_relative './capitalize_decorator'
+require_relative './trimmer_decorator'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] This should be within the "attr_accessor" and not the "attr_reader" thus making it something like this : ```attr_accessor :name, :age, :rentals``

Reply:
Done! Thanks